src/Components/Slideshow/Slideshow.css

```diff
@@ -17,3 +17,8 @@
 #clicker .carousel-indicators li{
   background-color: black;
 }
+
+.slideshow{
+  align-items: center;
+  height: 94vh;
+}
```

Comment:
add a blank newline at the bottom, [this post](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) explains why we do it